source/common/options.js

```diff
@@ -149,12 +149,23 @@ function buildOptionsPage() {
 
   settings.forEach(function (setting) {
     if (setting.type === 'checkbox') {
-      jq('#' + setting.section + 'SettingsPage').append(jq('<div>', { class: 'row option-row' }).append(jq('<input>', { type: 'checkbox', id: setting.name, name: setting.name, 'aria-describedby': setting.name + 'HelpBlock' })).append(jq('<div>', { class: 'option-description' }).append(jq('<label>', { for: setting.name, text: setting.title })).append(jq('<span>', { id: setting.name + 'HelpBlock', class: 'help-block', text: setting.description }))));
+      jq('#' + setting.section + 'SettingsPage')
+        .append(jq('<div>', { class: 'row option-row' })
+          .append(jq('<input>', { type: 'checkbox', id: setting.name, name: setting.name, 'aria-describedby': setting.name + 'HelpBlock' }))
+          .append(jq('<div>', { class: 'option-description' })
+            .append(jq('<label>', { for: setting.name, text: setting.title }))
+            .append(jq('<span>', { id: setting.name + 'HelpBlock', class: 'help-block' }))));
+      jq('#' + setting.name + 'HelpBlock').html(setting.description);
```

Comment:
Yes, I believe they'll accept it if it's gone through a type of processor that will prevent XSS first.

Probably the best way to get the formatting in there is by using template literals like this:

```javascript
    var reallyLongString = `
# Look at me

I can have spaces and all sorts of things.
`;
```

Or the other option would be to create a whitelist of HTML tags we accept and use something like this: https://github.com/leizongmin/js-xss